lib/jbuild

```diff
@@ -0,0 +1,11 @@
+(library
+ ((name ezxenstore)
+  (public_name ezxenstore)
+  (modules (xenstore watch))
+  (wrapped false)
```

Comment:
Yep, it's making namespacing (kind of) the default :tada: 
I think it should be always `true` (the default) if possible